### PR TITLE
Add Depth First Search in New Graph Theory Folder

### PR DESCRIPTION
**Pull request by sumitks866**

### CODE REVIEW

File: C++/Algorithms/Graph Theory Algorithms/depth_first_search.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		}
+	}
+}
+
+int main() 
+{
+	#ifndef ONLINE_JUDGE
+  freopen("input.txt","r",stdin);
+  freopen("output.txt","w",stdout);
+  #endif
```
Comment (abhpd) [inline C++/Algorithms/Graph Theory Algorithms/depth_first_search.cpp:29]: @sumitks866 Remove these lines, rest of the code is A-OK 👍

**Review by abhpd — CHANGES_REQUESTED** [review]
@sumitks866 Make the specified changes.

File: C++/Algorithms/Graph Theory Algorithms/depth_first_search.cpp (hunk 1 above)
Comment (sumitks866) [reply C++/Algorithms/Graph Theory Algorithms/depth_first_search.cpp]: Done

**Review by abhpd — APPROVED** [review]
@sumitks866 Thanks for contributing ❤️ Happy Hacktoberfest! 💫

**Pull request merged**